packages/compiler/server/serverlib.ts

```diff
@@ -531,6 +535,45 @@ export function createServer(host: ServerHost): Server {
     }
   }
 
+  async function addImportCompletion(
+    program: Program,
+    document: TextDocument,
+    completions: CompletionList
+  ) {
+    const new_path = getPath(document);
+    const projectroot = await findProjectRoot(compilerHost, new_path);
+    const new_file = await loadFile(
+      compilerHost,
+      resolvePath(String(projectroot), "package.json"),
+      JSON.parse,
+      program.reportDiagnostic
+    );
+    let dep_list: string[] = [];
+    for (const val of new_file) {
+      if (val.dependencies != undefined) {
+        dep_list = dep_list.concat(Object.keys(val.dependencies));
+      }
+    }
+    for (const val of dep_list) {
+      const update_projectroot = projectroot + "/node_modules/" + val;
+      const lib_fil = await loadFile(
+        compilerHost,
+        resolvePath(String(update_projectroot), "package.json"),
+        JSON.parse,
+        program.reportDiagnostic
+      );
+      for (const lib_fil_val of lib_fil) {
+        if (lib_fil_val.cadlMain != undefined) {
+          completions.items.push({
+            label: val,
+            kind: CompletionItemKind.Keyword,
```

Comment:
```suggestion
            kind: CompletionItemKind.Module,
```
I think module here might make more sense